disco.py

```diff
@@ -350,22 +401,26 @@ def createPath(filepath):
 
 if is_colab:
     if google_drive and not save_models_to_google_drive or not google_drive:
-        model_path = '/content/model'
+        model_path = '/content/models'
```

Comment:
I'm guessing you may have a reason why `models` may be preferred over `model`. Perhaps it's consistency with another project? I remember that I considered one better than the other for consistency with another project.. but I don't remember if I picked the right one in Disco. I hoped that I did.

Reply:
As discussed in DM on discord, It aligns better with JAX and i just think naming the folder as models just makes more sense since we have more than 1 to choose from.